src/Source.ml

```diff
@@ -41,6 +41,12 @@ let string_between t (l1 : Location.t) (l2 : Location.t) =
   then None
   else Some (String.sub t ~pos ~len)
 
+let empty_line_between t l1 l2 =
+  let non_whitespace _i c = not (Char.is_whitespace c) in
```

Comment:
nit: FWIW I tend to avoid names like `_i` since the code can still cheat and use it, and by itself `i` isn't so helpful for documentation. But as you choose.